src/tests/hubot-url-title.coffee

```diff
@@ -74,3 +74,19 @@ describe 'hubot-url-title', ->
         ['john', "http://cdimage.debian.org/debian-cd/8.3.0/amd64/iso-cd/debian-8.3.0-amd64-CD-1.iso"]
         ['hubot', "Resource at http://cdimage.debian.org/debian-cd/8.3.0/amd64/iso-cd/debian-8.3.0-amd64-CD-1.iso exceeds the maximum size."]
       ]
+
+
+  context "Japanese user posts link to Etsy", ->
+    beforeEach ->
+      nock('https://etsy.com', reqheaders: {'Accept-Language':'ja'}).get('/').replyWithFile(200, 'src/tests/test_files/etsy.ja.html')
```

Comment:
Etsy's title supports the internationalization like following, so I selected it as test data 🔶 

<img width="959" alt="2016-06-04 8 06 52" src="https://cloud.githubusercontent.com/assets/1257116/15795439/8de595e0-2a2c-11e6-90b4-681f24aa5474.png">